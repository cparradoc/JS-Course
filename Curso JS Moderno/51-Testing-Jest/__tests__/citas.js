import Citas from '../js/classes/Citas';


describe("Probar la clase de citas", () => {

    const citas = new Citas();

    test('Agregar una nueva cita', () => {
        const citaObj = {
            mascota: 'Hook',
            propietario: 'Parrado',
            telefono: '6666',
            fecha: '10/12/2022',
            hora:'10:30',
            sintomas: 'No se calla'
        };

        citaObj.id = Date.now();

        citas.agregarCita(citaObj);

        //Prueba
        expect(citas).toMatchSnapshot();



    });
});