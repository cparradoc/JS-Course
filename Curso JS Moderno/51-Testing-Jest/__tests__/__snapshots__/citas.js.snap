// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de citas Agregar una nueva cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "10/12/2022",
      "hora": "10:30",
      "id": 1648209161435,
      "mascota": "Hook",
      "propietario": "Parrado",
      "sintomas": "No se calla",
      "telefono": "6666",
    },
  ],
}
`;
